minecraft_servers/mcsucht/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "server_name": "mcsucht",
+  "nice_name": "MCSUCHT",
+  "direct_ip": "mcsucht.de",
+  "server_wildcards": [
+    "%.mcsucht.de"

```

Comment:
Doesn't look like a real wildcard
![image](https://github.com/LabyMod/server-media/assets/37186532/4cb2e6af-cda1-4d85-a288-728962e31ed1)

Reply:
Sorry, I removed the wildcard